app/population/controllers/weekly.js

```diff
@@ -0,0 +1,24 @@
+function dashboard(req, res) {
```

Comment:
Could we add a test for this controller? Even if it's just a simple unit test to check for the correct template, number of params and the correct params exist.

Reply:
Yeah, definitely - it's one of the things flagged up by CodeClimate (it's actually be helpful for this PR! :smile: )

Reply:
Tests added.